src/components/orchestrator/useRemote.tsx

```
@@ -6,7 +6,7 @@ function abortRequest() {
 	controller.abort();
 }
 
-function nothing() {}
+function nothing() { }
```

Why do you use nothing ? It makes errors fail silently.